diff --git a/configure/os/CONFIG.darwinCommon.darwinCommon b/configure/os/CONFIG.darwinCommon.darwinCommon
index 2c12b2fbb..5e39270cf 100644
--- a/configure/os/CONFIG.darwinCommon.darwinCommon
+++ b/configure/os/CONFIG.darwinCommon.darwinCommon
@@ -60,7 +60,7 @@ GNU = NO
 #
 # Darwin shared libraries
 #
-SHRLIB_LDFLAGS = -dynamiclib -flat_namespace \
+SHRLIB_LDFLAGS = -dynamiclib \
     -install_name $(abspath $(INSTALL_LIB))/$@ \
     $(addprefix -compatibility_version , $(SHRLIB_VERSION)) \
     $(addprefix -current_version , $(SHRLIB_VERSION))
